partner_contact_in_several_companies/views/res_partner.xml

```diff
@@ -1,149 +1,165 @@
-        </xpath>
-        <field name="is_company" position="after">
-            <field name="contact_type" invisible="1"/>
+    <record id="view_res_partner_tree_contact" model="ir.ui.view">
+        <field name="name">res.partner.tree.contact</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_tree"/>
+        <field name="arch" type="xml">
+            <field name="parent_id" position="after">
+                <field name="contact_id" invisible="1"/>
+            </field>
         </field>
-        <page name='internal_notes' position="before">
-            <page name="other_position" string="Other Positions" attrs="{'invisible': ['|',('is_company','=',True),('contact_id','!=',False)]}">
-                <field name="other_contact_ids" context="{'default_contact_id': active_id, 'default_name': name, 'default_street': street, 'default_street2': street2, 'default_city': city, 'default_state_id': state_id, 'default_zip': zip, 'default_country_id': country_id, 'default_supplier': supplier}" mode="kanban">
-                    <kanban create="false">
-                        <field name="color"/>
-                        <field name="name"/>
-                        <field name="title"/>
-                        <field name="email"/>
-                        <field name="parent_id"/>
-                        <field name="is_company"/>
-                        <field name="function"/>
-                        <field name="phone"/>
-                        <field name="street"/>
-                        <field name="street2"/>
-                        <field name="zip"/>
-                        <field name="city"/>
-                        <field name="country_id"/>
-                        <field name="mobile"/>
-                        <field name="state_id"/>
-                        <field name="image"/>
-                        <field name="lang"/>
-                        <templates>
-                            <t t-name="kanban-box">
-                                <t t-set="color" t-value="kanban_color(record.color.raw_value)"/>
-                                <div t-att-class="color + (record.title.raw_value == 1 ? ' oe_kanban_color_alert' : '') + ' oe_kanban_global_click'">
-                                    <a t-if="!read_only_mode" type="delete" class="fa fa-times pull-right"/>
-                                    <div class="o_kanban_image">
-                                        <img t-if="record.image.raw_value" t-att-src="'data:image/png;base64,'+record.image.raw_value"/>
-                                        <t t-if="!record.image.raw_value">
-                                            <img t-if="record.is_company.raw_value === true" t-att-src='_s + "/base/static/src/img/company_image.png"'/>
-                                            <img t-if="record.is_company.raw_value === false" t-att-src='_s + "/base/static/src/img/avatar.png"'/>
```

Comment:
company_image.png and avatar.png where moved to /base/static/img/

<img t-if="record.is_company.raw_value === true" t-att-src='_s + "/base/static/img/company_image.png"'/>
<img t-if="record.is_company.raw_value === false" t-att-src='_s + "/base/static/img/avatar.png"'/>